### ISSUE TITLE
Including labeled data into generative model

### ISSUE THREAD

**Issue by mhigginslp** [HazyResearch/snorkel#1075]
I would like to explore handing some of my "ground truth" labels to the generative model.  Can you provide some guidance?

In one of your introductory presentations, the presenter gave the suggestion that empirically it works better to use any labelled data to create LF's rather than give to generative model but I don't understand why it's bad practice to do both.  
 
Thanks!

**Comment by bhancock8** [comment]
If you have enough labeled data, yes you can provide both to the generative model. (You could, for example, treat your gold labeled data as being produced by another LF that happens to have perfect accuracy; or you could simply override any labels produced by the model with the gold standard labels that you know you have for those examples). However, in many of the applications where we've worked, labeled data is so valuable and expensive (e.g., labeled by a doctor whose time is very expensive) that you would rather use it for evaluation or guiding the production of LFs (which can be used to labels many thousands of points potentially) rather than using them as training data directly, since one you've included the labels in any way to train your model, they are no longer valid for evaluation.
